### ISSUE TITLE
Level of development stars

### ISSUE THREAD

**Issue by PMehrfeld** [RWTH-EBC/AixLib#365]
## What is the problem?

- `AixLib.UsersGuide.Conventions.ModelTemplateDocumentation` still recommend usage of the stars

## Why do we want to solve it?

- Not clear how the level of development is measured.

## How do we want to solve it?

- I want to collect some opinions here whether these stars should remain part of the template or not.

**Comment by AnaConstantin** [comment]
I personally always liked the star system and think the explanations for how many starts to give are clear. However as far as I remember the other Annex 60 participants were not so enthusiastic about it so it didn't become an Annex 60 standard. I vote that we remove it, otherwise it is a bit weird to have this inconsistency. A documentation template however is a very good idea, so in case the IPBSA / Annex 60 has none I suggest we keep ours, without the stars.,

**Comment by marcusfuchs** [comment]
I agree with @AnaConstantin - the stars are nice, but I would also remove them. Our documentation template (based on the [IBPSA/A60 template](https://github.com/ibpsa/modelica-ibpsa/wiki/Style-Guide#documentation)) is [in our Wiki on the *Documentation* page](https://github.com/RWTH-EBC/AixLib/wiki/Documentation#template). But it is down there at the bottom. Maybe it deserves its own page with 1 or 2 good examples?

**Comment by AnaConstantin** [comment]
I think we should also move the tempate in the User's Guide in AixLib. For me it makes the most sense to have it there, since you can just copy the template and fill it in. We already have two templates under AixLib.UsersGuide.Conventions, so I suggest removing them and adding the correct one.

**Comment by marcusfuchs** [comment]
@AnaConstantin You are right, I will open a new issue for that and put the template there.

**Comment by PRemmen** [comment]
Our goal is to delete all stars from AixLib documentation

**Comment by PMehrfeld** [comment]
Since it is not urgent, it's not my priority no. 1. But I'll take care of this.

**Comment by FWuellhorst** [comment]
I wrote a short python script to remove all stars. According to python and notepad++, 210 files had the "star"-description. The function i used just removed the lines containing "Level of Development" and "modelica://AixLib/Resources/Images/Stars/stars". I double checked the number of removed lines with the results i got from the notepad++ find in file option and the number i got searching with python.
I also deleted the AixLib/Resources/Images/Stars folder, because it is no longer used. 

https://github.com/RWTH-EBC/AixLib/tree/issue365_levelOfDevelopmentStars
